script.js

```diff
@@ -1,3 +1,49 @@
+/**
+ * AJAX request for users and groups
+ * Adapted from Struct plugin
+ *
+ * @param {function} fn Callback on success
+ * @param {string} id Call identifier
+ * @param {string} param Pass the parameter to backend
+ */
+function ajax_watchcycle(fn, id, param) {
```

Comment:
Can we wrap the whole code into a function to avoid polluting the global namespace?